devel/xapian-core/Portfile

```diff
@@ -179,7 +182,7 @@ foreach v {8.0 8.1 8.2} {
 }
 
 # Python
-foreach v {2.7 3.6 3.7 3.8 3.9 3.10 3.11} {
+foreach v {2.7 3.7 3.8 3.9 3.10 3.11} {
```

Comment:
are there dependents of the py27 subport - ideally we would get rid of that one as well.

Reply:
Done